### ISSUE TITLE
add custom footer icon

### ISSUE THREAD

**Issue by naveenjujaray** [wiredashio/wiredash-sdk#28]

**Comment by flschweiger** [comment]
Good idea, will add an option to customize the footer and branding icons 👍

**Comment by adiltirur** [comment]
Can we already change the footer icon? Is there an option already or can I can we change the image in the library?

**Comment by flschweiger** [comment]
This feature is planned for an upcoming release 🙂 However, currently this is not possible.
